pkg/controllers/report/resource/controller.go

```diff
@@ -320,7 +320,8 @@ func (c *controller) updateDynamicWatchers(ctx context.Context) error {
 		for _, policy := range vapPolicies {
 			kinds, err := admissionpolicy.GetKinds(policy.Spec.MatchConstraints, restMapper)
 			if err != nil {
-				return err
+				logger.Error(err, "failed to resolve kinds for ValidatingAdmissionPolicy", policy.Name)
```

Comment:
If we do this for all new policy types, why not extend it to clusterpolicy as well? Otherwise we would hace inconsistent behavior when kinds cannot be resolved?